frontend/public/components/utils/units.js

```diff
@@ -44,6 +44,11 @@ const TYPES = {
     space: true,
     divisor: 1000,
   },
+  cpu: {
+    units: ['mcores', 'cores'],
```

Comment:
/hold

These values *must* match the values that are used in the YAML for parsing values.

Reply:
`mcores` are not really a thing. I think this will be confusing. Either use `m` or spell out `milicores`.

Reply:
fixed to `millicores`